src/profile-logic/profile-data.js

```diff
@@ -1329,3 +1435,74 @@ export function getOriginAnnotationForFunc(
 
   return '';
 }
+
+/**
+ * Return a function that can compare two samples' call nodes, and determine a sort order.
```

Comment:
Add:

```
 * The order is determined as follows:
 *  - Ancestor call nodes are ordered before their descendants.
 *  - Sibling call nodes are ordered by their call node index.
 * This order can be different than the order of the rows that are displayed in the
 * call tree, because it does not take any sample information into account. This
 * makes it independent of any range selection and cheaper to compute.
```